public/site.webmanifest

```diff
@@ -48,8 +48,38 @@
             "type": "image/png"
         }
     ],
+    "shortcuts": [
+        {
+            "name": "Brew AeroPress",
+            "short_name": "AeroPress",
+            "description": "Brew a cup of coffee using the AeroPress",
+            "url": "/aeropress",
+            "icons": [
+                { "src": "/assets/icons/aeropress.png", "sizes": "256x256" }
+            ]
+        },
+        {
+            "name": "Brew French Press",
+            "short_name": "French Press",
+            "description": "Brew a cup of coffee using the French Press",
+            "url": "/french-press",
+            "icons": [
+                { "src": "/assets/icons/french-press.png", "sizes": "256x256" }
```

Comment:
Update the icon to have a red background and a white shape, as opposed to the black shape on the white background.